commons/all/src/main/java/org/infinispan/commons/configuration/attributes/Attribute.java

```diff
@@ -93,6 +97,10 @@ public boolean isModified() {
       return modified;
    }
 
+   public boolean isRepeated() {
+      return definition.isRepeated();
+   }
+
    public <K> K asObject() {
```

Comment:
what about some cleanup of this class:
*field `value` can be private (the class is final)
* methods `getValue`, `setValue`,  `setProtect`and `setModified` can be removed
* method `asObject` is used only a single place and ca be replace with `get`

Reply:
Yep